docker/latest/Dockerfile

```
@@ -40,6 +40,8 @@ RUN chmod +x /bin/bash_run \
     cmake \
     git \
     libfftw3-dev \
+    libopenblas-dev \
+    liblapack-dev \
```

```suggestion
    liblapacke-dev \
```

Maybe give a try to this package if you have problems with the headers